engine-tests/src/test/java/org/terasology/persistence/typeHandling/coreTypes/factories/EnumTypeHandlerFactoryTest.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2017 MovingBlocks
```

Comment:
Trivial: header template update for year :-)

Reply:
I forget to do a `./gradlew idea` every time I `pull upstream` :D Will the header automatically be updated for files I edit?